src/text_parser.py

```diff
@@ -0,0 +1,203 @@
+import re
+import nltk
+from nltk.corpus import words
+from nltk.stem import WordNetLemmatizer
+#import mistune
+import markdown
+'''preprocess data for input'''
+
+# remove previously referenced comments
+#remove every line that starts with ">"
+def remove_reference(text):
+    pattern = r'''^>.*'''
+    result = re.sub(pattern, "", text, 0, re.MULTILINE | re.IGNORECASE | re.VERBOSE)
```

Comment:
should be max 80 character width (http://google.github.io/styleguide/pyguide.html#32-line-length)

Reply:
Fixed.. need to run FormatCode on all .py files..